src/components/editors/exportEditor.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react'
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  DialogContentText,
+  Typography,
+} from '@material-ui/core'
+
+import makeCssFile from '../../lib/generate'
+import { loadStorage, loadCss } from '../../lib/utils'
+
+import { Button } from '../SettingComponent'
+
+const ExportEditor = () => {
+  const [ settings, setSettings ] = useState( false )
+
+  const saveFile = () => {
+    const link = document.createElement( 'a' )
+    link.setAttribute( 'download', 'customTheme.css' )
+    link.href = makeCssFile()
+    document.body.appendChild( link )
+    window.requestAnimationFrame( () => {
+      const event = new MouseEvent( 'click' )
+      link.dispatchEvent( event )
+      document.body.removeChild( link )
+    } )
+  }
+
+  const handleClickOpen = () => {
```

Comment:
btw you can one-line these ones:

`const handleClickOpen = () => setSettings(true)`